apmpackage/apm/0.1.0/agent/input/template.yml.hbs

```diff
@@ -2,3 +2,17 @@ apm-server:
     host: {{host}}
     secret_token: {{secret_token}}
     rum.enabled: {{enable_rum}}
+    max_event_size: {{max_event_bytes}}
+    capture_personal_data: {{capture_personal_data}}
+    api_key.enabled: {{api_key_enabled}}
+    api_key.limit: {{api_key_limit}}
+    rum.allow_origins: {{rum_allow_origins}}
+    rum.allow_headers: {{rum_allow_headers}}
+    rum.library_pattern: {{rum_library_pattern}}
+    rum.exclude_from_grouping: {{rum_exclude_from_grouping}}
+    rum.response_headers: {{rum_response_headers}}
+    rum.event_rate.limit: {{rum_event_rate_limit}}
+    rum.event_rate.lru_size: {{rum_event_rate_lru_size}}
+queue.mem.events: {{queue_mem_events}}
```

Comment:
Did you test that `queue.*` actually get picked up? I would not expect this to work, but I'm glad if it does.

Reply:
I'll remove the `queue` related pieces; created an issue for reloading the queue when configured for agent https://github.com/elastic/beats/issues/23993.